libp2p/crypto/rsa.nim

```diff
@@ -131,34 +137,38 @@ proc random*[T: RsaKP](t: typedesc[T], bits = DefaultKeySize,
   let pko = brRsaPrivateKeyBufferSize(bits)
   let eko = pko + brRsaPublicKeyBufferSize(bits)
 
+  var res: T
```

Comment:
```suggestion
  var res = new T
```
:)

but really, as noted elsewhere, the next refactoring is to turn this into non-ref probably

Reply:
It will be not possible to convert to non-ref, until you will not drop dependency on bearssl.